package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/ember-cli/broccoli-middleware",
   "dependencies": {
-    "handlebars": "^4.0.4",
+    "handlebars": "^4.0.13",
```

Comment:
To be clear, this version was already allowed.

Reply:
Yup 👍 it's just more likely to update the lock file I think.